.ruby-version

```diff
@@ -1 +1 @@
-2.7.7
\ No newline at end of file
+3.2.2
```

Comment:
Was there a particular reason to go to 3.2.2 instead of latest (3.3.3)?

Reply:
unfortunately the cloudflare builder is on 3.2.2, and updating it in `cloudflare.sh` didn't seem to work :/

Reply:
ah makes sense. sad